### PR TITLE
Update README documentation in UnifyCR repository

### PR DESCRIPTION
**Pull request by dsikich**
- Point top level README to full documentation on readthedocs
        - Add build status to top level README
        - Remove old build content from meta/README
        - Remove client/INSTALL and client/README

I plan to add the information about all of the issues in client/README into Github issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-## UnifyCR: A Distributed Burst Buffer File System - 0.1.0
+# UnifyCR: A Distributed Burst Buffer File System - 0.1.0
+
+## Build Status
+The current status of the UnifyCR dev branch is
+[!Build Status](https://api.travis-ci.org/LLNL/UnifyCR.png?branch=dev)]
```
Comment (nedbass) [inline README.md]: I think you're missing a bracket after the exclamation point:
```
[![Build Status](https://api.travis-ci.org/LLNL/UnifyCR.png?branch=dev)]
```

Comment (dsikich) [reply README.md]: @nedbass thanks, added now.

Comment (nedbass) [reply README.md]: Hmm the markdown is still not rendering quite right. Maybe the URL part needs to be on the same line.

Comment (dsikich) [reply README.md]: @nedbass okay, it looks like it is rendering correctly now.

**Pull request merged**